skimage/restoration/_denoise.py

```diff
@@ -64,6 +68,30 @@ def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
     >>> noisy = np.clip(noisy, 0, 1)
     >>> denoised = denoise_bilateral(noisy, sigma_range=0.05, sigma_spatial=15)
     """
+    if multichannel:
+        if image.ndim != 3:
```

Comment:
yeah I would say it's only implemented for 2D grayscale (image.ndim == 2) and 2D multichannel (image.ndim == 3) images. Something along those lines.